### DISCUSSION TITLE
issue with survey-likert plugin

### DISCUSSION THREAD

**Discussion by shaygr9** [jspsych/jsPsych#2651] [Q&A]
Hi,
I added a survey-likert element to my experiment:
![image](https://user-images.githubusercontent.com/72192563/171184512-e8ce8820-c7ee-4a68-8b0d-ad1370814fe5.png)

I have a problem that when I click somewhere around the survey it changes the chosen radio button to be the most left option ("1").

Does anyone have an idea how can I fix it?

Thanks in advance,
Shay Grunwald

**Comment by jodeleeuw** [comment]
Hi @shaygr9,

Are you using any additional CSS besides the `jspsych.css` file?

**Reply by shaygr9** [reply]
Hi @jodeleeuw , yes I am. is it a problem?

**Reply by jodeleeuw** [reply]
Maybe. It's likely that the additional CSS is somehow creating the issue since the standard CSS works OK as far as I know.

Do you have a link to your experiment that you can share?

**Reply by shaygr9** [reply]
I added the CSS for changing the font style and size of the scale.

I don't have the full project online yet, but here is a link for the survey files:
https://github.com/shaygr9/Boundries-Check

**Reply by shaygr9** [reply]
@jodeleeuw Hi again, I removed the CSS changes that I did and still have this problem.
I use jspsych-6.0.1, will it help to update the version? 
If yes, is there a way to update just the survey plugin?

**Reply by jodeleeuw** [reply]
Hi @shaygr9,

It looks like in these lines...

https://github.com/shaygr9/Boundries-Check/blob/e87c8d28d62cb47096d2642eb77bbf8dca1c7748/surprise_assessment.html#L95-L96

...the `<div>` and `<p>` tags are not closed at any point. That might be creating some unexpected behavior.
